wiki/BBCode/fr.md

```diff
@@ -1,395 +1,304 @@
-- Syntaxe :
-  - Où `HEXCODE` est un nom de couleur ou son code hexadécimal  Where `HEXCODE` is a colour in hexadecimal or colour name.
-    - Pour le code hexadécimal, il doit commencer par `#` suivit par 6 caractères hexadécimal (0 - 9, A - F).
-    - Pour le nom de couleur, il doit s'agir d'une couleur HTML valide.
-  ```
-  [color=HEXCODE] ... [/color]
-  ```
-
-Pour une liste du nom des couleurs, voyez [noms de couleur X11](https://fr.wikipedia.org/wiki/Noms_de_couleur_X11#Noms_de_couleur_identiques_entre_X11_et_HTML/CSS).
-
-### Taille de police
-
-Ajuste relativement la taille du texte.
-
-- Bouton dédié : ![option de taille de police](img/font-size.png)
-- Notes :
-  - Utilisez cette balise avec parcimonie.
-  - Vous pouvez utiliser uniquement 4 tailles :
-    - `50` (Très petite)
-    - `85` (petite)
-    - `100` (normale; par défaut)
-    - `150` (grande)
-  - Si vous entrez une valeur qui n'est pas une de celles listées ci-dessus, la valeur par défaut sera utilisée.
-  - L'argument `NUMBER` ne doit pas être entourés de guillemets.
-- Syntax :
-  - Où `NUMBER` est une des tailles suivantes (`50`, `85`, `100`, ou `150`) Where `NUMBER` is one of the listed sizes (`50`, `85`, `100`, or `150`).
-  ```
-  [size=NUMBER] ... [/size]
-  ```
+```
+[color=HEXCODE]texte[/color]
+```
+
+*Pour une liste de tous les noms de couleurs, voir [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart "Wikipédia (en anglais)")*
+
+La balise `[color]` est utilisée pour styliser le texte au moyen de différents types de couleurs sûres pour le web. Cette balise utilise le format [HEX code](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet "Wikipedia (en anglais)") pour spécifier la couleur, bien qu'elle puisse aussi être spécifiée par des noms de couleur HTML comme "rouge" ou "vert". Pour spécifier, remplacez l'argument `HEXCODE` par le code HEX ou le nom HTML correspondant à la couleur.
+
+L'argument mentionné ci-dessus ne prend pas de guillemets (`"`), et n'a pas de couleur par défaut. Si aucun argument n'est spécifié, ou si des guillemets sont utilisés, la balise ne sera pas analysée comme une balise BBCode.
+
+### Taille de la police
+
+```
+[size=NUMBER]texte[/size]
+```
+
+La balise `[size]` est utilisée pour styliser le texte en utilisant des tailles de police différentes. Actuellement, quatre tailles sont prises en charge par le site web d'osu! :  50, 85, 100 et 150. Ces tailles sont appelées respectivement "tiny", "small", "normal" et "large".
+
+L'argument `NUMBER` n'accepte pas les guillemets, et n'accepte qu'une des quatre tailles supportées. Si un nombre, qui n'est pas l'un des quatre supportés, est entré, le texte reviendra à la taille par défaut.
+
+Bouton de la barre d'outils : ![Font size options](img/font-size.png "Taille de la police")
 
 ### Spoiler
 
-*Ne pas confondre avec [spoiler déroulant](#spoiler-déroulant).*
+*À ne pas confondre avec [Spoilerbox](#spoilerbox).*
```

Comment:
```suggestion
*À ne pas confondre avec [Boîte à spoiler](#boîte-à-spoiler).*
```
It would probably be better to keep using the French name.
At the very least, make it so the link works by using `#boîte-à-spoiler` instead of `#spoilerbox`!